code/src/index.css

```diff
@@ -1,13 +1,221 @@
-body {
+/* Mobile version */
+* {
+  padding: 0;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  box-sizing: border-box;
+}
+
+body {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+  text-decoration-color: white;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.horizontalLine {
+  display: block;
+  color: #e9f4fe;
+  border-style: inset;
+  opacity: 0.2;
+}
+.main-container {
+  padding: 1em;
+}
+
+/* ------MainHeader------*/
+.mainHeading {
+  font-size: 25px;
+  color: white;
+  padding: 1em 0 1em;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.headerLogo {
+  width: 100px;
+}
+
+/* ----- ICONS -----*/
+
+.icons {
+  width: 40px;
+  z-index: 10;
+  position: absolute;
+  opacity: 0.5;
```

Comment:
Why opacity 0.5 here? I noticed that the icons were a bit faded. Maybe chenge this to opacity 1?

Reply:
Good pointer here from Linnea, I agree!